CONTRIBUTING.md

```diff
@@ -38,7 +37,7 @@ the [puppet-dev mailing list](https://groups.google.com/forum/#!forum/puppet-dev
 ## Making Changes
 
 * Create a topic branch from where you want to base your work.
-  * For new and active repositories, this is usually the main branch, but for
+  * For new and active repositories, this is usually the `main` branch, but for
     older or less active repos, the default branch may still be named `master`.
   * Only target release branches if you are certain your fix must be on that
```

Comment:
does the new backport workflow change this recommendation at all?

Reply:
I think it's ok as there could be a bug in code that only exists in the older branch, but was deleted in main. But maybe say "... target older branches ..." as I wouldn't expect contributors to PR a "release" branch.